nixos/services/mail/default.nix

```diff
@@ -86,6 +86,21 @@ in {
     let
       fqdn = with config.networking; "${hostName}.${domain}";
     in {
+      warnings = optional (role.imprintUrl != null || role.imprintText != null) ''
+        The imprintUrl or imprintText option needs to be set
+
+        This is mandated by:
+        - Telekom Postmaster Rules §4.1 (see https://postmaster.t-online.de/#t4.1), for sending mail to Telekom customers
+        - Telemediengesetz Anbieterkennzeichnung (see https://www.gesetze-im-internet.de/tmg/__5.html), for compilance with German law
+
+        Please do it as soon as possible
+      '';
+      assertions = [
+        { assertion = role.imprintUrl != null && role.imprintText != null;
```

Comment:
the assertion must be turned around: at the moment it fails when none or one of the options is set.